<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FixturesWindow</class>
 <widget class="QDialog" name="FixturesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fixtures</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10" stretch="2,1,0">
   <item>
    <widget class="QTableView" name="tblFixtures">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>10</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlChannelSliders"/>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>7</number>
     </property>
     <widget class="QCheckBox" name="chkUpdate">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Write On Change</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRefresh">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh Fixtures</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
